test/spec/change-streams/unified/change-streams-disambiguatedPaths.json

```diff
@@ -26,6 +26,7 @@
   "runOnRequirements": [
     {
       "minServerVersion": "6.1.0",
+      "maxServerVersion": "8.1.0",
```

Comment:
Since this isn't in the spec tests, do we have a drivers ticket to update them or do we plan on this being temporary? (In that case is there a Node ticket to revert?)

Reply:
https://mongodb.enterprise.slack.com/archives/GGWBN4ZNK/p1743452067317649?thread_ts=1743440886.743299&cid=GGWBN4ZNK